api/types/types.go

```diff
@@ -219,18 +222,34 @@ type InfoBase struct {
 	ContainerdCommit   Commit
 	RuncCommit         Commit
 	InitCommit         Commit
+	SecurityOptions    []string
 }
 
-// SecurityOpt holds key/value pair about a security option
 type SecurityOpt struct {
 	Key, Value string
 }
 
-// Info contains response of Remote API:
-// GET "/info"
-type Info struct {
-	*InfoBase
-	SecurityOptions []SecurityOpt
+func DecodeSecurityOptions(opts []string) ([]SecurityOpt, error) {
+	so := []SecurityOpt{}
+	for _, opt := range opts {
+		// support output from a < 1.13 docker daemon
+		if !strings.Contains(opt, "=") {
+			so = append(so, SecurityOpt{Key: "name", Value: opt})
+			continue
+		}
+		split := strings.Split(opt, ",")
```

Comment:
Pro-tip: use the `encoding/csv` parser to break down strings. It allows quoting and handles the commas nicely.

We use this format for the `mount` API and others with great effect.